content/composer/available-actions.md

```diff
@@ -6,42 +6,42 @@ draft: false
 weight: 300
 
 ---
+## Actions Bar
+On the top right you can find additional actions to perform on the current used blueprint.
 
-On the top right you can find additional actions to perform on the current opened blueprint.
-
-# Save
+### Save Blueprint
```

Comment:
The action bar button reads "Save", not "Save Blueprint", so I think the title should be "Save"